fsharp-backend/src/LibExecution/RuntimeTypes.fs

```diff
@@ -363,7 +367,11 @@ and DType =
 /// information later, such as the iteration count that led to this, or
 /// something like a stack trace
 and DvalSource =
+  /// Function was either called directly,
```

Comment:
What does "function was called directly" mean? Why wouldn't this have a SourceID?

Reply:
I was making a bit of a guess here - removing that part of the comment.